content/_includes/utility.macros.njk

```diff
@@ -199,16 +199,16 @@ params:
 ) %}
   {%- set ranges = {
     'same-y': {
-      'start': 'short_month',
-      'end': 'range'
+      'start': 'month',
+      'end': 'month_year'
     },
     'same-m': {
-      'start': 'month_date',
-      'end': 'date' if (format == 'month_date') else 'date_year'
+      'start': 'month_day',
+      'end': 'day' if (format == 'month_day') else 'day_year'
     }
   } -%}
 
-  {%- set ongoing = (format == 'range') and not end -%}
+  {%- set ongoing = (format == 'month_year') and not end -%}
```

Comment:
@jgerigmeyer changing range to month-year doesn't make a lot of sense to me. The purpose of that format is that it assumes a start and end date, and assumes the range is open-ended if there is no end date. "Range" seems much more accurate to that.